app/views/developers/_form.html.erb

```diff
@@ -60,20 +58,14 @@
               <span class="block text-sm font-medium text-gray-700">Cover image</span>
               <div class="relative">
                 <%= form.file_field :cover_image, direct_upload: true, accept: "image/png, image/jpg, image/jpeg, image/gif", data: {action: "input->file-upload#select direct-upload:initialize->file-upload#start direct-upload:error->file-upload#error"}, class: "hidden" %>
-                <%= form.label :cover_image, "Change", class: "cursor-pointer absolute right-0 bottom-0 z-10 m-5 bg-white py-2 px-3 border border-gray-300 rounded-md shadow-sm text-sm leading-4 font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500" %>
+                <%= form.label :cover_image, "Change", class: "cursor-pointer absolute right-0 bottom-0 z-10 m-5 bg-white py-2 px-3 border border-gray-300 rounded-md shadow-sm text-sm leading-4 font-medium text-gray-700 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500" %>
```

Comment:
Hey! Yeah, sorry this was probably off-topic for this PR. I noticed that because the button for cover image is physically on the cover image, the relative brightness of the hover state is way lower than the button on the white background. This was so much the case, that I originally thought the hover-state had been left off entirely. I made this change since I felt that it brought the hover states to the same relative color.